modules/infinite-scroll/themes/twentyseventeen.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Infinite Scroll Theme Assets
+ *
+ * Register support for Twenty Seventeen.
+ */
+
+/**
+ * Add theme support for infinite scroll
+ */
+function twentyseventeen_infinite_scroll_init() {
```

Comment:
hey @Stoyan0v! this all looks great and I'd really like to merge this today to make it into the next release. 

My only request is to please name space these functions with `jetpack_` to eliminate any possibility of redeclaration.  Thanks!